### PR TITLE
Homepage partners

### LINKED ISSUE #3: 🎨 HomePage Partners Section
#  🎨 Create the partner's section on the HomePage
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a section that shows/lists all our core business partners. Check the full design below for section placement. Use the assets provided below.

partners-section

Please take note of all the design features

#### Link to full design:
```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```

#### Company Logos
VVOB:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/vvob-logo_qyvjbp.jpg
```

KSA:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/Kenya_Scouts_Association_kcoka5.svg
```

farmbetter:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/farmbetter-logo_x2acda.png
```
WWF:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380406/elewa-group-website/client-logos/wwf-4-logo-png-transparent_tbkfcq.png
```

Enabel:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380405/elewa-group-website/client-logos/Enabel-logo_afiel7.png
```

Spotlights:
```
https://res.cloudinary.com/dyl3rncv3/image/upload/v1681380412/elewa-group-website/client-logos/Spotlights_7_pplnvm.jpg
```

### Technical Analysis
This section should be present in the `index.html` file

### Acceptance Criteria
1. When rendered the component is pixel-perfect compared to the design above.
2. The component is clean and does not have more than 50 lines
3. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Pychangeric**
# This pull request up holds the work for the partners section where I have added the partners logos has required by the checks. THis is the up
dated promt for the screen to be responsive.

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #3 (issue#3) - *Example*: Fixes #10, #20 NOTE: GitHub issue numbers should be prefixed with **#** to aid with issue tracking.

issue #3 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional) 
image

image
image

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,11 +9,89 @@
...
...b5YGyFz+3Xbuk2aKRCW3aApTn0u4TEyFqmAZRl6Ewp3+0Vz6m3Qoe5V8eb96DpfwxbEwDhWlDNkNi7VeRp/+ZC7dfVuZ/joKbS7BQ0iCQ0vR4SFjWNVkfzKXSZ/JV3IZFRhhHWCGy5Wzd5SeHXxVi6zQY+3sgLNcRrYfHAQVl/Bv+UD8H3b8uF+9OJdVn0cUhiB2Gf9SBocxf1BbwrplILuBNiLI71dxKf04TwyoBvqJZeNx8LVk9hmuq7cutiQuoyW+3uxB9hlnKo0wYTv+p8x+IdkN2QfnKl4Jq15HAXlX7j3fehZu08YwQ9+qW+CjqMukGL5l3RnCZbSwMkooe6dlCik8SScgBEZY/BFCr6tqDbCay+7BhRl2J2Zwl7lc5cO51jK+srisNJLlXdzNSMcL49xJEho1+ANpCxShE+d9xbSqnsPxdWeVefIORc5G2OchTzw939N7ByVt+XjBtaWo8NSWzBVgPDujxnq5nCVWpTlZCiCJxuFquVR2BPRmy6XiVR/dSXThmaIBZ7hybTjA1n0lUPUbuET6diXCmsBjCe9nlxc+0301hM9r+q/yfNZaPx6lM5JyF41LtEnln0SDFLukfcxrMG/9JBtM4vJ8+++/BOnp5SOJ1YYz5rLEqSPdB26VNbXvx4IkXHtbfNNJFQlphY99TVvTzCiMYB5ek3IX9NKpvx0h4RWDaBnpjZC+5MEVvepJoNYncJklhfLuMn5Jf+2vxk1AkrBjyewRPNPAuZr3PKUYrgi7eHDWToVs2J3k59Qd++kqN0q6oewu49j8+Q9g5nq5R1/zTjSEXY8gfyJBMFZXvkNSzm5cOgj0Mn//IKpndonTgdlRDqDavVCLSXZ3tYryGNiumjowO3AeouUVeP9/4FxT+J2P4RPldesYnP7UKMoThPVL1Ca3764r+CbiRXsfH6Mo1Wj4WmrzqjdGyUW1R/fdfXd63Rv25CK8o6lNztpGUeaiRvA2r3enHl10+Xm1yb3Xq6kzFsb92e0izYaQemi2VSkPh82NotTEzRKrr3FmNtC9BO9+JlPsWmZj5wsxOvY2fddsOH45mis7iKoT3PFcs6NzMYSLPxYTeK23jMkpjvths2nSagYGBgYGBgYGBgYGBgYGBgYGBgYGBgYGBgYGBgb/C/4DtGXOohcr7C4AAAAASUVORK5CYII=" alt="savics"/>
```
Comment (JAPHETHNYARANGA) [inline index.html:75]: ?? why

File: src/css/style.css [hunk 2]
```diff
@@ -1,5 +1,157 @@
...
+    }
+}
+.header
+{
+    display:inline-flex;
+}
+.logo img 
+{
+    margin-left: 20px;
+    margin-top: 30px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:68]: use a single margin property for this

File: src/css/style.css [hunk 3]
```diff
@@ -1,5 +1,157 @@
...
+    text-align: center;
+    color: #2D2E52;
+    text-decoration: none;
+    padding: 10px ;
+    font-size: 20px;
+}
+.con button {
+    float: right;
+    margin-top: 50px;
+    margin-left: 450px;
```
Comment (JAPHETHNYARANGA) [inline src/css/style.css:95]: this also.  If the margin property has four values:

margin: 25px 50px 75px 100px;
top margin is 25px
right margin is 50px
bottom margin is 75px
left margin is 100px

**Review by JAPHETHNYARANGA — CHANGES_REQUESTED** [review]
Please make the page responsive and send screenshots of web and mobile view

**Comment by Pychangeric** [comment]
@JAPHETHNYARANGA   Hello , I have done the collection on the respinsiveness. I ask for Re-review

**Comment by JAPHETHNYARANGA** [comment]
closing this pr as issue is done already, please pick another issue

**Pull request closed without merging**